### DISCUSSION TITLE
Inf values

### DISCUSSION THREAD

**Discussion by gogogoglo** [dayoonkwon/BioAge#3] [Q&A] [answered]
Hi there! After generating estimated biological ages using the phenoage_calc function, I was unable to calculate basic summary statistics for because there were some infinity (“Inf”) values present in the phenoage variable. Could you advise on why this might happen and how to best address?

**Comment by dayoonkwon** [comment, marked as answer]
Inf values were created because of some extreme values from the biomarker data. PhenoAge algorithm requires a similar range of biomarkers. I recommend removing any outliers. If you have FEV or other large unit variables as one of your biomarkers, I recommend rescaling it (e.g., FEV/1000 mL) so the value is not very different from other biomarkers.
